unime/src/routes/welcome/password/confirm/+page.svelte

```diff
@@ -12,12 +14,23 @@
   import Smiley from '~icons/ph/smiley';
   import SmileySad from '~icons/ph/smiley-sad';
 
-  let passwordsEqual: boolean | undefined;
+  // 3 states: true (match), false (mismatch), undefined (I don't know).
+  let passwords_match: boolean | undefined = undefined;
+
   let showPassword = false;
+  let password = '';
+
+  // Ref to input DOM element.
+  let input_element: HTMLInputElement;
+
+  onMount(() => {
+    // Initial focus.
```

Comment:
I don't think we need this comment as the code explains itself.

Reply:
Comment removed.